packages/docmanager-extension/src/index.ts

```diff
@@ -257,6 +260,18 @@ function addCommands(app: JupyterLab, docManager: IDocumentManager, palette: ICo
     mnemonic: args => args['mnemonic'] as number || -1
   });
 
+  commands.addCommand(CommandIDs.openTab, {
+    execute: args => {
+      const path = typeof args['path'] === 'undefined' ? ''
+        : args['path'] as string;
```

Comment:
I think it is probably better to do nothing here if path is not defined, or if it is the empty string.

Reply:
Good point, I added a check to handle this case.